src/app_server.cr

```diff
@@ -9,6 +11,7 @@ class AppServer < Lucky::BaseAppServer
       Lucky::ErrorHandler.new(action: Errors::Show),
       Lucky::RouteHandler.new,
       CacheControlHandler.new,
+      Lucky::StaticCompressionHandler.new("./public", file_ext: "gz", content_encoding: "gzip"),
```

Comment:
I'm thinking all 3 of these could be Habitat settings instead

Reply:
I don't think these should be habitat settings. I think if you were to make changes you would probably add brotli, but in that case you still need the gzip fall through.

```crystal
Lucky::StaticCompressionHandler.new("./public", file_ext: "br", content_encoding: "br"),
Lucky::StaticCompressionHandler.new("./public", file_ext: "gz", content_encoding: "gzip"),
```

Reply:
Oh interesting. So you could have multiple StaticCompressionHandlers that handle different file types?

Reply:
If sot hen yeah I totally agree with you and that's very cool!

Reply:
Exactly.

Just add
```js
    plugins: [
      webpackNotifier,
      new CompressionPlugin({
        compressionOptions: { level: 9 },
        test: /\.js$|\.css$|\.html$|\.svg$/
      }),
      new CompressionPlugin({
        compressionOptions: { level: 11 },
        filename: '[path].br[query]',
        algorithm: 'brotliCompress',
        test: /\.js$|\.css$|\.html$|\.svg$/
      })
    ],
```
to your webpack.mix.js and you're rolling.

Reply:
That is sooo cool. I love it